test/AspNet.Security.OAuth.Providers.Tests/Alipay/AlipayTest.cs

```diff
@@ -54,5 +54,10 @@ public async Task Can_Sign_In_Using_Alipay(string claimType, string claimValue)
                 AssertClaim(claims, claimType, claimValue);
             }
         }
+
+        private sealed class FixedClock : ISystemClock
+        {
+            public DateTimeOffset UtcNow => new DateTime(2019, 12, 14, 22, 22, 22, DateTimeKind.Local);
```

Comment:
Sorry, my mistake here to use the wrong type in the snippet I gave you. It should be:

```csharp
public DateTimeOffset UtcNow => new DateTimeOffset(2019, 12, 14, 22, 22, 22, TimeSpan.Zero);
```

Reply:
It's my mistake, I set "timestamp" .ToLocalTime(),  so you tests has not pass. I will change all to utc. it will be ok.